scripts/ops/openrouter_route.py

```
@@ -0,0 +1,231 @@
+    }
+    req = urllib.request.Request(
+        url,
+        data=json.dumps(body).encode("utf-8"),
+        headers={
+            "Authorization": f"Bearer {api_key}",
+            "Content-Type": "application/json",
+            "HTTP-Referer": "https://github.com/alawein/alawein",
+            "X-Title": "alawein-control-plane",
+        },
+        method="POST",
+    )
```

Ruff (0.16.2)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)